wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -132,6 +137,23 @@ class SwiftGenerator private constructor(
       else -> null
     }
 
+  // see https://protobuf.dev/programming-guides/proto3/#default
+  private val Field.proto3InitialValue: String
+    get() = when {
+      isMap -> "[:]"
+      isRepeated -> "[]"
+      isOptional -> "nil"
+      isEnum -> "." + enumType.constants.first().name
+      else -> when (typeName.makeNonOptional()) {
+        BOOL -> "false"
+        DOUBLE, FLOAT -> "0"
+        INT32, UINT32, INT64, UINT64 -> "0"
+        STRING -> "\"\""
+        DATA -> ".init()"
+        else -> "nil"
```

Comment:
Definitely not saying it's wrong, but I feel a little itchy that you're deriving a lot of logic on this "nil".

Reply:
Yeah, was kind of feeling the same. There should be an `isOptional` property on most fields I believe.

Reply:
Moved control flow to rely on `isOptional` instead